### DISCUSSION TITLE
Features/tweaks compared to Sophia Script

### DISCUSSION THREAD

**Discussion by adamhl8** [Sophia-Community/SophiApp#177] [General]
Hi there,

I'm curious how SophiApp compares to Sophia Script in terms of features/available tweaks. Does SophiApp do everything Script does?

Also, is the plan to eventually stop updating Script in favor of SophiApp?

**Comment by farag2** [comment]
Hi.
Consider Sophia Script as a locomotive of both two projects. Firstly, the script gets new feature, then it is adopted for the app. But now the active development of SophiApp has stopped due to is ha the EOL (end of life) status from us. All efforts are directed towards SophiApp 2.0 (https://t.me/SophiaNews/1311). No ETA yet. So for now we're consider adding new function, probably bug fixing only.
The answer for you question: SophiApp covers 95% of functions in the Script. So the next aim for us is to move all current and upcoming ones from the script to the new app. Unfortunately, we couldn't and didn't want to itfor the first branch.

> Also, is the plan to eventually stop updating Script in favor of SophiApp?

Never. I do love PowrShell and automatisation. Sophia Script is my own project, unlike SophiApp: when we work together on it with my pal. He's the main dev, not me. I have another responsibilities. :) Read more here: 

* https://t.me/SophiaNews/746
* https://habr.com/en/post/683452/

Feel free to ask anything. 🐱

**Reply by adamhl8** [reply]
Makes sense. Thanks!
